src/lib/viewers/BaseViewer.js

```diff
@@ -344,7 +344,10 @@ class BaseViewer extends EventEmitter {
         // Add a resize handler for the window
         document.defaultView.addEventListener('resize', this.debouncedResizeHandler);
 
-        this.addListener('load', () => {
+        this.addListener('load', (event) => {
+            // this.scale set to 1 if event.scale does not exist
+            ({ scale: this.scale = 1 } = event);
```

Comment:
I know that's not how it works now, but wouldn't it make more sense if set scale set `this.scale` as well as set the attribute on the annotated element?

If so, then move to 

```
const { scale } = event.scale;
//...
this.annotator.setScale(scale);
```
and `setScale()` can be

```
setScale(scale = 1) { 
  this.scale = scale;
  // ... set attribute and stuff
}
```